### ISSUE TITLE
Add "disable" option for accessories

### ISSUE THREAD

**Issue by rpolenthon** [hoobs-org/HOOBS#469]
It would be useful for users with big and complex setups, to be able to disable single accessories, so they get ignored by Homebridge and still remain in the config.

**Why do you need this?**
If you work with MQTT accessories, you sometimes decide for example to add some accessories just during the Christmas period, the rest of the year they would be useless in HomeKit. The problem: If you delete the accessories from the config, you have to configure every accessory again next year. A disable option would remove it from HomeKit and still keep the configuration information in the config, so the next year you only have to enable the accessories and everything is in HomeKit again.

This feature request has no high priority, but in the future, it could be a useful enhancement.

**Comment by donavanbecker** [comment]
Wouldn’t it loose its config in the home app though?

**Comment by rpolenthon** [comment]
@donavanbecker Yes, that's the whole point. Sometimes you may want to remove the accessories from home, but keep the config parts. (And you don't want to store them separately in notes) ;)
